standards/brokerapi.proto

```diff
@@ -0,0 +1,174 @@
+syntax = "proto3";
+
+import "google/protobuf/struct.proto";
+
+service SpiffeBrokerAPI {
+    /////////////////////////////////////////////////////////////////////////
+    // X509-SVID Profile
+    /////////////////////////////////////////////////////////////////////////
+
+    // Fetch X.509-SVIDs for all SPIFFE identities the referenced workload is 
+    // entitled to, as well as related information like trust bundles. As this
+    // information changes, subsequent messages will be streamed from the server.
+    rpc FetchX509SVID(X509SVIDRequest) returns (stream X509SVIDResponse);
+
+    // Fetch trust bundles of the referenced workload. Useful in situations that
+    // only need to validate SVIDs without obtaining an SVID for themself. As this
+    // information changes, subsequent messages will be streamed from the server.
+    rpc FetchX509Bundles(X509BundlesRequest) returns (stream X509BundlesResponse);
+
+    /////////////////////////////////////////////////////////////////////////
+    // JWT-SVID Profile
+    /////////////////////////////////////////////////////////////////////////
+
+    // Fetch JWT-SVIDs for all SPIFFE identities the referenced workload is 
+    // entitled to, for the requested audience. If an optional SPIFFE ID is 
+    // requested, only the JWT-SVID for that SPIFFE ID is returned.
+    rpc FetchJWTSVID(JWTSVIDRequest) returns (JWTSVIDResponse);
+
+    // Fetches the JWT bundles, formatted as JWKS documents, keyed by the
+    // SPIFFE ID of the trust domain. As this information changes, subsequent
+    // messages will be streamed from the server.
+    rpc FetchJWTBundles(JWTBundlesRequest) returns (stream JWTBundlesResponse);
+}
+
+// The WorkloadPIDReference message conveys a process id reference of a workload 
+// running in the same environment.
+message WorkloadPIDReference {
```

Comment:
In K8s when containerd is used as the container runtime, a pod has a 'pause' container and then the workload containers. We can have 2 or more pids for example (I don't think crio has a pause container anymore) can it be clarified which pid would be used in this case? I know in the spire delegated identity, if you pass the pause pid it fails attestation. It would be great to have this explicitly stated? Maybe I just need to go back and read the original documents related to that effort and missed it.